torchtitan/config/job_config.py

```diff
@@ -97,6 +97,18 @@ class Metrics:
     """Whether to log metrics to Weights & Biases"""
 
 
+@dataclass
+class ExtraLosses:
```

Comment:
This section is specifically for MoE load balancing loss for now, do you foresee any other loss related params will be used in this section? If not, let's make the name for descriptive and specific